### PR TITLE
Fix tag UI changes

### PR DESCRIPTION
**Pull request by psav**
{{pytest: cfme/tests/cloud_infra_common/test_tag_objects.py -k test_tag_item_through_details}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3f161298210d794b26e84a342aa6ec113cbc9a7e:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/configure/configuration/__init__.py [hunk 1]
```diff
@@ -114,18 +114,24 @@ def invalidate_server_details():
         ('name', Input("name")),
         ('display_name', Input("description")),
         ('description', Input("example_text")),
-        ('show_in_console', Input("show")),
-        ('single_value', Input("single_value")),
-        ('capture_candu', Input("perf_by_tag"))
+        ('show_in_console', CFMECheckbox("show")),
+        ('single_value', CFMECheckbox("single_value")),
+        ('capture_candu', CFMECheckbox("perf_by_tag"))
```
Comment (jkrocil) [inline cfme/configure/configuration/__init__.py]: So... these dont need to be version picked?

Comment (psav) [inline cfme/configure/configuration/__init__.py]: No - it's a SUPER Widget :)

Comment (jkrocil) [inline cfme/configure/configuration/__init__.py]: Fair enough. Merging.

Comment (psav) [inline cfme/configure/configuration/__init__.py]: Thanks :)

File: cfme/configure/configuration/__init__.py [hunk 2]
```diff
@@ -114,18 +114,24 @@ def invalidate_server_details():
...
 tag_form = Form(
     fields=[
         ('category', Select("select#classification_name")),
         ('name', Input("entry[name]")),
         ('display_name', Input("entry[description]")),
-        ('add', Input("accept")),
-        ('new', "//span[@class='glyphicon glyphicon-plus']")
+        ('add', {
+            version.LOWEST: Input("accept"),
+            '5.6': '//button[normalize-space(.)="Add"]'
```
Comment (jkrocil) [inline cfme/configure/configuration/__init__.py]: Actually one more thing - https://github.com/RedHatQE/cfme_tests/blob/master/cfme/web_ui/form_buttons.py#L122

Comment (cbx33) [inline cfme/configure/configuration/__init__.py]: Fair enough

**Pull request merged**